src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -140,7 +140,7 @@ private function renderException(FlattenException $exception, string $debugTempl
             'exceptionMessage' => $exceptionMessage,
             'statusText' => $statusText,
             'statusCode' => $statusCode,
-            'logger' => DebugLoggerConfigurator::getDebugLogger($this->logger),
+            'logger' => class_exists(DebugLoggerConfigurator::class) ? DebugLoggerConfigurator::getDebugLogger($this->logger) : null,
```

Comment:
@radar3301 suggested a small optimization that makes sense to me:

```suggestion
            'logger' => ($this->logger && class_exists(DebugLoggerConfigurator::class)) ? DebugLoggerConfigurator::getDebugLogger($this->logger) : null,
```

Reply:
I added a check for the property not being `null`